src/docs/integration/linux/build-tools/buildroot.md

```diff
@@ -0,0 +1,80 @@
+# Buildroot
+
+You can integrate the Peridio Agent into buildroot by leveraging the [buildroot external tree](https://buildroot.org/downloads/manual/manual.html#outside-br-custom).
+
+Checkout the peridio external tree from GitHub: [peridio/buildroot-external-peridio](https://github.com/peridio/buildroot-external-peridio)
```

Comment:
Checkout the peridio external tree from [GitHub](https://github.com/peridio/buildroot-external-peridio).